pom.xml

```diff
@@ -56,8 +58,109 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>ssh-slaves</artifactId>
-            <version>1.25</version>
+            <version>1.27</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>structs</artifactId>
+            <version>1.17</version>
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <profiles>
+        <profile>
+            <id>docker-generate-binaries</id>
+            <activation>
+                <os>
+                    <family>unix</family>
+                </os>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
```

Comment:
BTW there is also a Maven plugin to run Docker (or probably a bunch of them), which may or may not be helpful.

Reply:
yeah,  i was debating whether to use them or not. I think because this is less common case where I'm building an image but not pushing it, I'll put all the commands in a script so someone can better see what's going on.